### ISSUE TITLE
Question on Test Accuracy being reported

### ISSUE THREAD

**Issue by mtmarsh2** [NVIDIA/DIGITS#57]
I trained a model, and set aside 25% of the data for test purposes, but after I trained the model I didn't see any reporting of the test accuracy. I noticed the blue line for accuracy, but thats training accuracy right? Does digits report the test accuracy for a test set?

**Comment by lukeyeager** [comment]
If you hover over the graph, it will give you more details about each line. As you can see, "accuracy" is a **validation** output ("val"). By default, DIGITS runs a pass through the validation set after one pass through the training set. If you want to change that, you can provide a different value for "Validation Interval" when creating your model.

**Comment by mtmarsh2** [comment]
Cool that makes sense, so is there a point where the model reports the final test accuracy after completion?

**Comment by lukeyeager** [comment]
Not yet, but it's on the agenda. See #17.

**Comment by mtmarsh2** [comment]
Cool, I'll close this issue then
